.gitlab/build_toss4.yml

```diff
@@ -40,7 +40,7 @@ toss4-clang_14_0_6-src:
     EXTRA_CMAKE_OPTIONS: "-DENABLE_BENCHMARKS=ON"
     DO_INTEGRATION_TESTS: "yes"
     ALLOC_NODES: "2"
-    ALLOC_TIME: "30"
+    ALLOC_TIME: "45"
```

Comment:
I would suggest also changing the deadline option from 1hour to 90minutes (on line 5). That way, jobs can sit on the queue a bit longer.

`SCHEDULER_PARAMETERS: "--res=ci --exclusive=user --deadline=now+1hour -N ${ALLOC_NODES} -t ${ALLOC_TIME} -A ${ALLOC_BANK}"`

to

`SCHEDULER_PARAMETERS: "--res=ci --exclusive=user --deadline=now+90minutes -N ${ALLOC_NODES} -t ${ALLOC_TIME} -A ${ALLOC_BANK}"`